### ISSUE TITLE
SP9-item2 : documentation review and tuning

### ISSUE THREAD

**Issue by srcansiz** [fedbiomed/fedbiomed#115]
In GitLab by @mvesin on Sep 17, 2021, 15:08

From the initial user documentation rewrite :
* review documentation
* correct, tune, make homogeneous

Tasks (to be completed if needed):
- [x] [all] propose colors for fedbiomed logo & vote from JLS' logo .svg
- [x] [ML] big picture review to standardize/homogeneize the outlines
- [x] [ML] add page "what's fedbiomed" (redundancy with below ?)
- [x] [ML] add page "about us" with
         1. what is fedbiomed 
         2. who is contributing?
         3. ongoing collaboration with companies
- [x] [ML] add "contact us" content
- [x] [ML] populate first "news" entries
- [x] [?] on main page, add "join us" subsection: "if you are interested in joining, contributing, ..., please contact Marco"
- [x] [SC] fonts homogeneize, encoding (accents)
- [x] [SC] top bar menus (Home / User documentation / About / Acknowledgement / Publications / News / Contact us) + side bar menus for doc (Getting started / Tutorials / User guide / Developer) + bottom (add News and Contact us)
- [x] [YB] perceptron accuracy check (compare local)
- [x] [JS] implementing local model for sklearn (tutorials)
- [x] [YB] implemeting local model to pytorch (tutorials)
- [x] [JS] adding realistic datasets for SGD regression
- [x] [SC] rename "Quick start" to "Let's start"
- [x] [MV] rename "Basic installation" "Installation" (explaining single machine)
- [x] [IB] install video & add to software installation page
- [x] [all] review all doc, suggest modifications (via discord, tomorrow daily)
- [x] [all] review all doc, correct directly typos/misc/python highlighting in `feature/review`
- [x] [SC] add table of contents to docs page, fix docs sidebar for mobile view, add highlighting functionality
- [x] [SC] complete 'deploying datasets' documentation
- [x] [SC] create google account (fedbiomed email) for google analytics
